opm/simulators/timestepping/SimulatorTimer.hpp

```diff
@@ -20,7 +20,6 @@
 #ifndef OPM_SIMULATORTIMER_HEADER_INCLUDED
 #define OPM_SIMULATORTIMER_HEADER_INCLUDED
 
-#include <opm/input/eclipse/Schedule/Schedule.hpp>
 #include <opm/simulators/timestepping/SimulatorTimerInterface.hpp>
```

Comment:
in that case we can drop the pointer, the pointer is used to avoid pulling in the boost headers in other compile units.

Reply:
>> I think we should probably include `<boost/date_time/gregorian/gregorian_types.hpp>` here.
>
> in that case we can drop the pointer, the pointer is used to avoid pulling in the boost headers in other compile units.

Understood.  I think we should include the Boost header here.

Reply:
okay, done, pointer usage dropped.